### PR TITLE
Add minimal travis support.

### PR DESCRIPTION
**Pull request by mstemm**
Add minimal travis.yml file that builds and packages falco. No actual
tests yet.

### CODE REVIEW

**Comment by mstemm** [comment]
@henridf

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - sudo apt-get --force-yes install g++-4.8
+    - sudo apt-get install rpm linux-headers-$(uname -r)
+    - git clone https://github.com/draios/sysdig.git ../sysdig
+before_script:
+    - export KERNELDIR=/lib/modules/$(ls /lib/modules | sort | head -1)/build
+script:
+    - set -e
+    - export CC="gcc-4.8"
+    - export CXX="g++-4.8"
+    - wget https://s3.amazonaws.com/download.draios.com/dependencies/cmake-3.3.2.tar.gz
```
Comment (henridf) [inline .travis.yml]: just curious why take cmake here and not from standard package repos?

Comment (mstemm) [inline .travis.yml]: This was what sysdig was doing. Part of the reason might be that the travis distro has an older version of CMake available. I remembered that on centos 7 I had to install a newer version.

Comment (henridf) [inline .travis.yml]: arright

**Comment by henridf** [comment]
👍

**Pull request merged**